interface/orders/procedure_provider_edit.php

```diff
@@ -70,8 +70,8 @@ function invalue($name)
 
 </head>
 
```

Comment:
The ? symbol that appears when you hover over a field and allows you to click for more information does not work at all.
![image](https://user-images.githubusercontent.com/46497068/91786412-8f1faa80-ebd5-11ea-9903-a251e93f7416.png)

Reply:
Update the commit.
![image](https://user-images.githubusercontent.com/22230889/92089657-09804280-ee01-11ea-8458-70b5085ab7ad.png)